libtmux/server.py

```diff
@@ -142,7 +142,7 @@ def _list_sessions(self):
         sformats = formats.SESSION_FORMATS
         tmux_formats = ['#{%s}' % f for f in sformats]
 
-        tmux_args = ('-F%s' % '\t'.join(tmux_formats),)  # output
+        tmux_args = ('-F%s' % '$@$'.join(tmux_formats),)  # output
```

Comment:
@JonathanRaiman I created https://github.com/tmux-python/libtmux/discussions/355 to discuss a default formatter string